### DISCUSSION TITLE
Is possible access tables in different mysql schemas?

### DISCUSSION THREAD

**Discussion by carlosfiori** [prisma/prisma#3110] [Q&A] [answered]
I need to access tables that are in a different scheme than the one specified in the connection.

I tried to map the table name using @@map, but it didn't work.

```
model User {
  id Int @id
  email String @unique
  name String?
  @@ map ("schema2.User")
}
```

It is always using the schema defined in the connection getting: blog.schema2.User

Is there a way to specify to use another schema?

**Comment by janpio** [comment]
What is a schema in MySQL context for you?

In general right now Prisma only connects to 1 schema (Postgres) or database (MySQL, SQLite) and you can not combine these into one Prisma Client.

**Comment by carlosfiori** [comment]
Another database in the same server, same connection

**Reply by ryands17** [reply, marked as answer]
Hey @carlosfiori 👋 
Unfortunately this isn't supported by Prisma currently and there is an issue [here](https://github.com/prisma/prisma/issues/2443) for the same.
You can also check [this](https://github.com/Errorname/prisma-multi-tenant) community project if it suits your needs.

**Reply by carlosfiori** [reply]
I will look at this project, thank you.

Who build the query that will be executed? Is it the prism or query-engine? I ask to take a look and maybe contribute to it.

**Reply by ryands17** [reply]
I think it just manages the different schemas that you have and thier endpoints, the query execution is done by Prisma's query engine itself.

**Reply by carlosfiori** [reply]
My problem is that I have relations between two schema (or db), something like this:

```
dbUser:
  tableUsers
  tablePreferences
dbPost:
  tablePosts
  tablePostsFiles
```

tableUsers having many tablePosts.
I need make a query like this:
```
{
  tableUsers {
    id
    name
    tablePosts {
      id
      title
    }
  }
}

```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
